pkg/operator/starter.go

```diff
@@ -127,6 +127,19 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		controllerConfig.EventRecorder,
 	)
 
+	cv, err := configInformers.Config().V1().ClusterVersions().Lister().Get("version")
```

Comment:
One should wait until `configInformers.Config().V1().ClusterVersions().Informer().HasSynced` is true before accessing the object. Just to be sure to avoid NotFound error.

Reply:
Done.